src/Framework/FieldsAPI/Concerns/HasVisibilityConditions.php

```diff
@@ -75,11 +75,15 @@ public function failsVisibilityConditions(array $values): bool
     /**
      * Set a condition for showing the node.
      *
+     * @unreleased Fixed a bug with encoded HTML entities in the operator, ie > (greater than), < (less than)
      * @since 2.27.3 update to use SimpleConditionSet
      * @since 2.13.0
      */
     public function showIf(string $field, string $operator, $value, string $boolean = 'and'): self
     {
+        // Fixes a bug with encoded HTML entities in the operator, ie > (greater than), < (less than)
+        $operator = html_entity_decode($operator);
```

Comment:
This might be better placed elsewhere, but this is the first attempt.

Reply:
Hmm. So what, exactly, is sending the html-entity form of the operator? FFM?

Reply:
It looks like the `$operator` is being encoded in the preview data sent over the wire from the builder.

Reply:
I'd trace it back and either prevent it from being encoded when sent, or decode it on the receiving end before it's used in this function.

Reply:
It looks like the `$operator` is being encoded in the preview data sent over the wire from the builder.

Reply:
Actually, I think this might be the best place for it, since the data is so nested. This is the central point at which the operator is passed to the validation rules. Otherwise, we'd have to cover multiple entry points to deeply decode HTML entities for this specific case.

Reply:
Let's talk about this more next week. I feel like this is an encroachment of responsibility. The Field API (and especially conditions) shouldn't have the foggiest clue as to what an HTML entity even is. It's removed from that layer. If it's multiple points to correct elsewhere, we can refactor in another way, but I don't think this is the right place to do it, simply the easiest at this point.